docs/quickstarts/powerpoint-quickstart-content.md

```diff
@@ -34,22 +34,28 @@ In this article, you'll walk through the process of building a PowerPoint [conte
 
 ### Update the code
 
-1. **Home.html** specifies the HTML that will be rendered in the add-in's task pane. In **Home.html**, find the `<p>` element that contains the text "This example will read the current document selection." and replace the entire element with the following markup then save the file.
+1. **Home.html** specifies the HTML that will be rendered in the add-in's task pane. In **Home.html**, find the `<p>` element that contains the text "This example will read the current document selection." and the `<button>` element where the id is "get-data-from-selection". Replace these entire elements with the following markup then save the file.
```

Comment:
Not an essential change, but I think wrapping "id" in code ticks makes this more readable. 
```suggestion
1. **Home.html** specifies the HTML that will be rendered in the add-in's task pane. In **Home.html**, find the `<p>` element that contains the text "This example will read the current document selection." and the `<button>` element where the `id` is "get-data-from-selection". Replace these entire elements with the following markup then save the file.
```